platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -184,22 +138,15 @@ public static void launch(@NonNull final CommandLineArgs commandLineArgs, final
      * Launch the browser but do not handle any exceptions
      *
      * @param commandLineArgs the parsed command line arguments
-     * @param pcesRecovery if true, the platform will be started in PCES recovery mode
+     * @param pcesRecovery    if true, the platform will be started in PCES recovery mode
      */
     private static void launchUnhandled(@NonNull final CommandLineArgs commandLineArgs, final boolean pcesRecovery)
             throws Exception {
         Objects.requireNonNull(commandLineArgs);
 
-        StartupTime.markStartupTime();
-        logger.info(STARTUP.getMarker(), "\n\n" + STARTUP_MESSAGE + "\n");
-        logger.debug(STARTUP.getMarker(), () -> new NodeStartPayload().toString());
-
-        // This contains the default PathsConfigs values, since the overrides haven't been loaded in yet
-        final PathsConfig pathsConfig = loadPathsConfig();
-
         // Load config.txt file, parse application jar file name, main class name, address book, and parameters
         final ApplicationDefinition appDefinition =
-                ApplicationDefinitionLoader.loadDefault(pathsConfig.getConfigPath());
+                ApplicationDefinitionLoader.loadDefault(getAbsolutePath("./config.txt"));
```

Comment:
isn't `"./config.txt"` the same as `"config.txt"` ? If yes I would prefer the second one, but maybe that is just me :)

Reply:
They are equivalent. I'm ok leaving out the `./`. Change made.